### PR TITLE
doc: Rename natEquals to _≡ᵇ_

### PR DESCRIPTION
**Pull request by Lysxia**
It's its name in `Data.Nat.Base`.

### CODE REVIEW

**Comment by Lysxia** [comment]
huh the PDF build doesn't like unicode. that's a shame.

**Comment by gallais** [comment]
Isn't it just a matter of adding the missing declarations in this file?
https://github.com/agda/agda/blob/master/doc/user-manual/unicode-symbols-sphinx.tex.txt

**Comment by andreasabel** [comment]
Rendering at:  https://agda--6352.org.readthedocs.build/en/6352/language/instance-arguments.html#declaring-instances

Maybe the underlying problem was that `natEquals` is somewhat undefined in the eyes of the documentation reader, because its import is hidden.  This problem persists with the renaming to `_≡ᵇ_ `, and I'd say it is rather worse because from `natEqual` you can rather straightforwardly guess its meaning, that it is an equality function for `Nat`, which isn't true any more for `_≡ᵇ_`.

Another thing is that the page `instance-arguments` has a predominantly ASCII style for identifiers, so the unicode does not fit well here.

Is `natEquals` not an understandable name?  If so, maybe `_==Nat_`?

File: doc/user-manual/language/instance-arguments.lagda.rst [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
   open import language.built-ins
     using (Bool; true; false; List; _∷_; []; Nat; _-_; zero; suc; _+_)
-    renaming (_==_ to natEquals)
+    renaming (_==_ to _≡ᵇ_)
```
Comment (andreasabel) [inline doc/user-manual/language/instance-arguments.lagda.rst:8]: See my comments in the PR conversation.

**Review by andreasabel — CHANGES_REQUESTED** [review]

**Comment by Lysxia** [comment]
In this case it occurs exactly once, in the RHS of `eqNat`, so I think the context makes it quite unambiguous. Or how about adding the provenance as a comment next to it? I think if a reader wants to try this code, since the imports are not rendered, they will guess some modules to import, and if they guess `Data.Nat` then `_≡ᵇ_` will just work, unlike `natEquals`.

**Comment by andreasabel** [comment]
> Or how about adding the provenance as a comment next to it?

This might be a solution!

**Review by andreasabel — APPROVED** [review]
Fine!

**Pull request merged**